mmaction/models/backbones/cuda_shift/setup.py

```diff
@@ -19,7 +34,7 @@
 else:
     raise ValueError('CUDA is not available')
 
-extra_compile_args = {'cxx': []}  # '-fopenmp', '-std=c99']
+extra_compile_args = dict(cxx=[])  # ['-fopenmp', '-std=c99']
```

Comment:
what does the commented flags mean? and why they are comments?

Reply:
Unused compilation switch